ReactiveCocoaTests/Swift/TestError.swift

```diff
@@ -14,3 +16,6 @@ enum TestError: Int {
 
 extension TestError: ErrorType {
 }
+
+class TestNSError: NSError {
```

Comment:
I thought the same thing, but when I checked the docs to verify that, they seem to actually _encourage_ it.

> NSError is not an abstract class, and can be used directly. Applications may choose to create subclasses of NSError to provide better localized error strings by overriding localizedDescription.

Regardless, it sounds like it's at least a bad practice, so it's probably better to test the common case here, like you suggested. So I just removed `TestNSError` and updated the tests.